ui/pages/developpeurs.js

```diff
@@ -17,34 +19,30 @@ const developpeurs = () => (
     <Navigation />
     <Breadcrumb forPage="developpeurs" label="Développeurs" />
 
-    <div className="c-page-container container my-0 mb-sm-5 p-5">
-      <div className="row">
-        <div className="col-12 col-md-5">
-          <h1>
-            <span className="d-block c-page-title is-color-2">Développeurs</span>
-          </h1>
+    <Container p={12} my={0} mb={[0,12]} variant="pageContainer">
+      <Grid templateColumns="repeat(12, 1fr)">
+        <GridItem px={4} colSpan={[12,12,12,5]}>
+          <Text variant="editorialContentH1" as="h1">Développeurs</Text>
           <hr className="c-page-title-separator" align="left" />
```

Comment:
to remove or change to Chakra-UI : https://chakra-ui.com/docs/components/divider